src/Spec/SubSpec.php

```
@@ -10,75 +10,114 @@
         return $this->filter;
     }
 
     /**
-     * Returns the default failure message for this rule specification.
-     *
-     *
-     * @access protected
-     * @return mixed[][]
+     * Returns the default failure message: the sub-filter's field messages.
      */
     protected function getDefaultMessage(): array
```

PHPStan (2.1.54)

[error] Return type (array) of method Aura\Filter\Spec\SubSpec::getDefaultMessage() should be compatible with return type (string) of method Aura\Filter\Spec\Spec::getDefaultMessage()


(method.childReturnType)